File: 100-days-of-code/src/day_04/IsJohnnyMakingProgress.java
package day_04;

import challengeInterface.Challenge;

public class IsJohnnyMakingProgress implements Challenge{
    
    int progressDays(int[] milesruned){
        int progressDaysCount = 0;

        for (int i = 0; i < milesruned.length; i++) {
            if (i == 0) {
                continue;
            } else if (milesruned[i - 1] < milesruned[i]) {
                progressDaysCount += 1;
            }
        }
        return progressDaysCount;
    }

    @Override
    public void run() {
        int[] progress = {9, 9};
        System.out.println(this.progressDays(progress));
    }
}

File: 100-days-of-code-csharp/Day04/IsJohnnyMakingProgress/IsJohnnyMakingProgress.cs
public class IsJohnnyMakingProgress
{
    public int ProgressDays(int[] milesRunned)
    {
        int progress = 0;
        for (int i = 0; i < milesRunned.Length; i++)
        {
            if (i == 0)
            {
                continue;
            }
            else if (milesRunned[i - 1] < milesRunned[i])
            {
                progress += 1;
            }
        }
        return progress;
    }
}